src/main/resources/json/court-details.json

```diff
@@ -3,474 +3,474 @@
     "courtName": "Bromley County Court And Family Court",
     "courtAddress": "Bromley County Court, College Road, Bromley, BR1 3PX",
     "phoneNumber": "0208 290 9620",
-    "email": "family.bromley.countycourt@justice.gov.uk"
+    "email": "fr_applicant_sol@sharklasers.com"
```

Comment:
what's happening here? is this temporary?

Reply:
Yeah temporary for Vivek/Me to test using this email, as we dont actually want to send this to an FRC just yet

Reply:
but we're not merging this right? did you add the toggle to send to this test address in nonprod envs?

Reply:
toggle has now been added!